src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/InstanceOwnerLookup.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Storage.Interface.Models;
+
+namespace Altinn.Platform.Storage.Models
+{
+    /// <summary>
+    /// Model to hold a instance owner lookup element
+    /// </summary>
+    [Serializable]
+    public class InstanceOwnerLookup
+    {
+        /// <summary>
+        /// Person number (national identification number)
+        /// </summary>
+        [JsonProperty(PropertyName = "personNumber")]
```

Comment:
Is there a reason we are not using SSN or socialSecurityNumber? This is used in the register component

Reply:
Comment from Skatt. They claim ssn is an american concept and that we shuld use national identification number instead. PersonNumber is somewhat in between.